### PR TITLE
Add HTTP Toolkit to members.csv

### PR DESCRIPTION
**Pull request by pimterry**
Adding HTTP Toolkit to the members list :smiley:

The README says "Include links to your branding materials" - unclear exactly what materials you'd like, but here's a selection:

* [High res logo with transparent background](https://github.com/user-attachments/assets/fd572f77-e80f-4a16-b4f2-43879afcd920)
* [Logo + text with transparent background](https://github.com/user-attachments/assets/2af626f6-94e7-49de-9165-2a03ea135ac0)
* [Full brand book](https://github.com/user-attachments/files/16512994/Http.Toolkit.-.Brand.Guidelines.pdf)

~~This doesn't match the current website yet, there's a pending brand update that I'll be rolling out in just a few weeks which tweaks things in a few places, but it'll all match up nicely soon. I have no problem with these being published & visible in places before that's live, but if you'd prefer current materials in the meantime let me know.~~

### CODE REVIEW

**Comment by vladh** [comment]
Thank you for this @pimterry! So cool to have you as the first member. Let me work out a CI issue and I'll get this merged. :)

**Comment by vladh** [comment]
@chadwhitacre To unblock this PR, we need to remove the `main` branch protections on the repo, because they prevent the GitHub runner itself from updating the member JSON files. Silly, but that's apparently how it works.

In any case, the protections are actually not required. The same result can be achieved by only giving push access to the repo to the people currently allowed to override the protections, and requiring everyone else to create a PR based on a fork, which anyone can do anyway.

**Comment by chadwhitacre** [comment]
Huzzah! 🥳 👏 @pimterry 

@vladh Do you have the permissions you need to make the GitHub configuration changes to unblock this PR? If not let's get you them. :-)

Also it looks like we've got a conflict now on this PR, unfortunately.

**Comment by vladh** [comment]
@chadwhitacre Nope, I don't have the permissions.

I can fix the conflict, no trouble.

**Comment by chadwhitacre** [comment]
Try now?

**Review by chadwhitacre — CHANGES_REQUESTED** [review]
Hey we're talking internally about reporting requirements and such, let's hold off on merging this for a minute while we play out this conversation ... I'll return here soon.

**Comment by vladh** [comment]
@chadwhitacre No worries! I've fixed the technical issues in the meantime.

**Comment by dcramer** [comment]
Talking live with chad, but I opened a ticket for more transparency #45

**Comment by vladh** [comment]
Hey @pimterry, thanks so much for your patience while we ironed some things out.

I've now updated [the README](https://github.com/opensourcepledge/osspledge.com) with the latest version of our reporting requirements.

The main change, I think, is that we expect members to have some kind of blog post on their website explaining the donations they made to open source projects. Ideally, but not strictly necessarily, this would have payments itemised by recipient where possible, and I've created [a tool to help with this for GitHub sponsors payments](https://github.com/opensourcepledge/osp-github-reporter/).

When you have the time, if you could make a blog post then update this PR that would be amazing. ✨ 

Happy to hear your feedback on this!

**Comment by chadwhitacre** [comment]
> I've now updated [the README](https://github.com/opensourcepledge/osspledge.com) with the latest version of our reporting requirements.

... aaaaaaaaand this has been moved to [osspledge.com/join](https://osspledge.com/join/). 😬 🙇‍♂️

**Comment by pimterry** [comment]
> When you have the time, if you could make a blog post then update this PR that would be amazing. ✨
> 
> Happy to hear your feedback on this!

I'll try to find some time to put something together for this soon, but I have a lot going on right now and it's going to take a little while to put this together.

That said, I hear the arguments but I'm still not a huge fan of this as a requirement - I'd much rather link contributions to the platforms themselves where possible (more verifiable, and better for those platforms too) and lower barriers.

This is also something that ends up a little weird for organizations who've been donating to open source for a while, if they don't already have these annual reports written & published. It's quite odd to now publicly publish a new "Here's the open source we donated to in 2021" report in 2024, but it seems the intent is that it won't be possible to include those past contributions without doing so.

Anyway, I'll work on putting something together next week if I can, watch this space.

**Comment by vladh** [comment]
I do get that the blog post requirement introduces friction. It does seem to me that the blog post could be very minimal — a couple of paragraphs and a list of payments.

**Comment by dcramer** [comment]
@pimterry you dont need to publish in retro (I wouldn't bother at the very least), but without that level of accountability (and the marketing gained from it) the program can't really succeed. If it was just a way to say "here are companies that use open collective" its kind of a waste of time. We really want to basically create almost this social peer pressure amongst organizations where we can overwhelm them with the message, create a sort of FOMO/shame situation if they don't participate, especially if they say they care.

**Review by vladh — APPROVED** [review]
Love your post @pimterry! 🔥

**Comment by pimterry** [comment]
I've now written up a nice blog post with a proper breakdown at https://httptoolkit.com/blog/open-source-funding-pledge/, updated the JSON linked here (https://httptoolkit.com/data/oss-pledge.json) to the latest format, and rebased on top of the other updates.

I do still mildly disagree on these requirements (agree on peer pressure & accountability goals, sure, but being able to show the largest possible number of peers seems like the key to peer pressure, and there are easy routes to accountability for the common case that don't add friction or exclude orgs who'd contribute to that peer pressure). That said, it's your project, you can set the rules as you'd like - I hope the feedback is useful, but obviously I'm still very much in favor of the whole enterprise in general regardless, it's certainly not a deal breaker.

**Comment by chadwhitacre** [comment]
Thank you for working with us on this, @pimterry! 🙏

**Review by chadwhitacre — APPROVED** [review]

**Pull request merged**
